collection_requirements.rst

```diff
@@ -189,6 +189,22 @@ Also:
 
 See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ for complete details.
 
+Deprecation, redirection, and removing of moved content
+-------------------------------------------------------
+
+If the collections are parts of Ansible distribution:
+
+1. Copy content from the source collection to the target collection.
+2. Deprecate the module/plugin with ``removal_version`` scheduled for the next major version in ``meta/runtime.yml`` of the source collection. If the target collection has been released with the new content as a new minor version for the same major version that is currently included in Ansible, this ensures that users can resolve the deprecation warning without having to install something manually that potentially breaks backwards compatibility.
+3. When the next major release comes:
+
+  * remove the module/plugin from the source collection
+  * add ``redirect`` to the corresponding entry in ``meta/runtime.yml``
+  * remove ``removal_version`` from there
+
+4. In 1-2 years, revisit the redirect.
```

Comment:
I'm not sure whether this is something that every collection maintainer should do, or whether it's something the community (meeting) should do.

Reply:
do you mean the whole procedure of text or exactly this points? We the procedure in general, we could add something like "moving a module/plugin from c.g./c.n...."

Reply:
if in general, anyway better to commit what was decided yesterday. I think the checklist is appropriate place for that (maybe there is better one, i don't know).

Reply:
Good question, I'm not 100% what exactly we meant with this yesterday. Whether we just want to revisit the redirects we added to c.g/c.n, or all / the procedure in general.

Reply:
let's remove `revisit` then

Reply:
done